### PR TITLE
Update

### PR DESCRIPTION
**Pull request by GuilhermeGuimaraesDeMedeiros**
task05-1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "onboarding-servlerless-cdk-students",
+  "name": "Serverless_Flavio",
```
Comment (flaviostutz) [inline package.json]: Why did you change this name?

**Review by flaviostutz — COMMENTED** [review]
It seems ok. Nice that you attached the screenshot with the script run results :)
I made just one question.

File: package.json (hunk 1 above)
Comment (GuilhermeGuimaraesDeMedeiros) [reply package.json]: I don't remember clearly, but I think it happened when I tried to clone the forked repository. I still have some difficulties working with GitHub. First, I choe the directory "onboarding-servlerless-cdk-students", and something went wrong. Then I chose another directory, which I named "Serverlees_Flavio".

Comment (flaviostutz) [reply package.json]: I understood it. But even if renaming the directory you don't need to change this attribute name. You can undo this change :)

**Review by flaviostutz — CHANGES_REQUESTED** [review]
Pls undo the package name update

**Comment by GuilhermeGuimaraesDeMedeiros** [comment]
Task5

**Comment by flaviostutz** [comment]
Please delete node_modules from the branch and add node_modules .gitignore to avoid it to be commited accidentally to the repo again.

node_modules dir is automatically generated and shouldn't be in the repo.

**Review by flaviostutz — CHANGES_REQUESTED** [review]
Please rename the "name" attribute in package.json to the original value

**Review by flaviostutz — APPROVED** [review]
Niceee!
